internal/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -64,16 +64,17 @@ ENV OPERATOR=/usr/local/bin/ansible-operator \
 # yum clean all && rm -rf /var/yum/cache/* first
 RUN yum clean all && rm -rf /var/cache/yum/* \
  && yum -y update \
- && yum install -y python36-devel gcc python3-pip python3-setuptools \
+ && yum install -y libffi-devel openssl-devel python36-devel gcc python3-pip python3-setuptools \
```

Comment:
Just curious why these are being added?

Reply:
Some Python extensions (e.g. cffi) have "manylinux" wheels for x86 but not for other arches, resulting in them needing to be built from source.  These were the additional dependencies I encountered when working on implementing this PR.